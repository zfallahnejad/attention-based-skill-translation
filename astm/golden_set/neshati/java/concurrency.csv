179850,concurrency
192444,concurrency
829571,concurrency
571407,concurrency
22656,concurrency
103154,concurrency
522444,concurrency
992484,concurrency
3474,concurrency
2711488,concurrency
276052,concurrency
331052,concurrency
1827903,concurrency
122207,concurrency
183406,concurrency
21234,concurrency
2071828,concurrency
230513,concurrency
14955,concurrency
