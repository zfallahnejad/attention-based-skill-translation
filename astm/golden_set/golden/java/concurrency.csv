179850,109,81,0.5736842105263158
192444,83,99,0.45604395604395603
829571,60,58,0.5084745762711864
571407,60,50,0.5454545454545454
22656,57,27,0.6785714285714286
103154,26,16,0.6190476190476191
522444,23,25,0.4791666666666667
992484,19,13,0.59375
3474,19,20,0.48717948717948717
2711488,18,21,0.46153846153846156
276052,16,10,0.6153846153846154
331052,14,5,0.7368421052631579
1827903,14,17,0.45161290322580644
122207,13,17,0.43333333333333335
2071828,12,10,0.5454545454545454
21234,12,8,0.6
183406,12,9,0.5714285714285714
230513,11,15,0.4230769230769231
14955,11,14,0.44
265143,10,11,0.47619047619047616
157882,10,12,0.45454545454545453
